src/main/java/com/commercetools/sync/commons/utils/AssetReferenceReplacementUtils.java

```diff
@@ -18,20 +18,19 @@
 
     /**
      * Takes an asset list that is supposed to have all its assets' custom references expanded in order to be able to
-     * fetch the keys and replace the reference ids with the corresponding keys for the custom references. This method
-     * returns as a result a {@link List} of {@link AssetDraft} that has all custom references with keys replacing the
-     * ids.
+     * fetch the keys for the custom references. This method returns as a result a {@link List} of {@link AssetDraft}
```

Comment:
```suggestion
     * get the keys for the custom references. This method returns as a result a {@link List} of {@link AssetDraft}
```

Reply:
`fetch` means go somewhere and get it (https://www.google.com/search?q=define%20fetch). I would use it for methods where you need to fetch the resource from CTP, but here it doesn't do that. I saw lots of methods use fetch in this context, so it's up to you if you want to fix it.

Reply:
Thanks, I would note this is not the only place that we have this misusage.